trade-finance-manager-ui/templates/case/underwriting/pricing-and-risk/_macros/section-exporter/exporter-table.njk

```diff
@@ -1,100 +1,137 @@
+        classes: 'govuk-tag--yellow',
+        attributes: {
+          'data-cy': 'exporter-table-credit-rating-not-added'
+        }
+      }) }}
+    {% endset %}
+    {% set editExporterCreditRatingActionText = 'Add' %}
+  {% endif %}
+
+  {% if userCanEditGeneral %}
+    {% set creditRatingRow = {
+      key: { html: exporterCreditRatingHeadingHtml },
+      value: { html: exporterCreditRatingValueHtml },
+      actions: {
+        items: [
+          {
+            href: "/case/" + caseId + "/underwriting/pricing-and-risk/edit",
+            text: editExporterCreditRatingActionText,
+            attributes: {
+              "data-cy":"exporter-table-credit-rating-action-link"
+            }
+          }
+        ]
+      }
+    } %}
+  {% else %}
+    {% set creditRatingRow = {
+      key: { html: exporterCreditRatingHeadingHtml },
+      value: { html: exporterCreditRatingValueHtml }
+    } %}
+  {% endif %}
+
+  {% set lossGivenDefaultHeadingHtml %}
+    <span data-cy="exporter-table-loss-given-default-heading">Loss given default</span>
+  {% endset %}
+
+  {% if lossGivenDefault %}
+    {% set lossGivenDefaultValueHtml %}
+      <span data-cy="exporter-table-loss-given-default-value">{{ lossGivenDefault }}%</span>
+    {% endset %}
+  {% else %}
+    {% set lossGivenDefaultValueHtml %}
+      <span data-cy="exporter-table-loss-given-default-value">-</span>
+    {% endset %}
+  {% endif %}
+
+  {% if userCanEditGeneral %}
+    {% set lossGivenDefaultRow = {
+      key: { html: lossGivenDefaultHeadingHtml },
+      value: { html: lossGivenDefaultValueHtml },
+      actions: {
+        items: [
+          {
+            href: "/case/" + caseId + "/underwriting/pricing-and-risk/loss-given-default",
+            text: "Change",
+            attributes: {
+              "data-cy":"exporter-table-change-loss-given-default-link"
+            }
+          }
+        ]
+      }
+    } %}
+  {% else %}
+    {% set lossGivenDefaultRow = {
+      key: { html: lossGivenDefaultHeadingHtml},
+      value: { html: lossGivenDefaultValueHtml }
+    } %}
+  {% endif %}
+
+  {% set probabilityOfDefaultHeadingHtml %}
+    <span data-cy="exporter-table-probability-of-default-heading">Probability of default</span>
+  {% endset %}
+
+  {% if probabilityOfDefault %}
+    {% set probabilityOfDefaultValueHtml %}
+      <span data-cy="exporter-table-probability-of-default-value">Less than {{ probabilityOfDefault }}%</span>
+    {% endset %}
+  {% else %}
+    {% set probabilityOfDefaultValueHtml %}
+      <span data-cy="exporter-table-probability-of-default-value">-</span>
+    {% endset %}
+  {% endif %}
+
+  {% if userCanEditGeneral %}
+    {% set probabilityOfDefaultRow = {
+      key: { html: probabilityOfDefaultHeadingHtml },
+      value: { html: probabilityOfDefaultValueHtml },
+      actions: {
+        items: [
+          {
+            href: "/case/" + caseId + "/underwriting/pricing-and-risk/probability-of-default",
+            text: "Change",
```

Comment:
`Change` context is clear for people with sight, but for screen readers we should add `visuallyHiddenText` or `<span class="govuk-visually-hidden">`. 
 
 Requirement example - https://ukef-dtfs.atlassian.net/browse/DTFS2-6467